README.md

```diff
@@ -36,6 +36,7 @@
 > There are two contributers for this project and we both eqully contributed
 > Here is screenshoot of the kanban board
 >
+
```

Comment:
- [ ] This command won't work:

![image](https://github.com/getssh/Doctors_appointment/assets/57421823/b3dea4f9-e038-417b-8461-3b4b95486d1c)

This is because this command is intended for the frontend tests.  But this is the backend repo readme.

Therefore, since you are not including backend tests, kindly remove this section. 👍